packages/vscode-extension/src/handlers.ts

```diff
@@ -1987,6 +1999,7 @@ export async function migrateTeamsTabAppHandler(): Promise<Result<null, FxError>
     }
     ExtTelemetry.sendTelemetryEvent(TelemetryEvent.MigrateTeamsTabApp, {
       [TelemetryProperty.Success]: TelemetrySuccess.Yes,
+      [TelemetryProperty.UpdateFailedFiles]: updateFailedFiles.length.toString(),
```

Comment:
Will you just send the number of failed files instead of file names?

Reply:
Yes, one reason is that the file paths will be blocked by GRPR like `<REDACTED: user-file-path>`, another reason is that the filenames will not contribute any help for us to troubleshooting or BI report.